.github/workflows/milestone.yml

```diff
@@ -0,0 +1,83 @@
+name: milestone
+
+on:
+  push:
+    branches: [master]
+
+defaults:
+  run:
+    shell: bash --noprofile --norc -euo pipefail {0}
+
+jobs:
+  set-milestone:
+    if: github.repository_owner == 'trinodb'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Get milestone from pom.xml
+        run: |
+          .github/bin/retry ./mvnw -v
```

Comment:
retry script doesn't exist in this repository.

Reply:
will add ..